mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -393,37 +393,54 @@ function is_rosetta_site() {
 function get_global_menu_items() {
 	$global_items = array(
 		array(
-			'title' => esc_html_x( 'Plugins', 'Menu item title', 'wporg' ),
-			'url'   => 'https://wordpress.org/plugins/',
-			'type'  => 'custom',
-		),
-
-		array(
-			'title' => esc_html_x( 'Themes', 'Menu item title', 'wporg' ),
-			'url'   => 'https://wordpress.org/themes/',
-			'type'  => 'custom',
-		),
-
-		array(
-			'title' => esc_html_x( 'Patterns', 'Menu item title', 'wporg' ),
-			'url'   => 'https://wordpress.org/patterns/',
-			'type'  => 'custom',
+			'title'   => esc_html_x( 'News', 'Menu item title', 'wporg' ),
+			'url'     => 'https://wordpress.org/news/',
+			'type'    => 'custom',
 		),
-
 		array(
-			'title' => esc_html_x( 'Learn', 'Menu item title', 'wporg' ),
-			'url'   => 'https://learn.wordpress.org/',
-			'type'  => 'custom',
+			'title'   => esc_html_x( 'Download & Extend', 'Menu item title', 'wporg' ),
+			'url'     => 'https://wordpress.org/download/',
+			'type'    => 'custom',
+			'submenu' => array(
+				array(
+					'title' => esc_html_x( 'Themes', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/themes/',
+					'type'  => 'custom',
+				),
+				array(
+					'title' => esc_html_x( 'Patterns', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/patterns/',
+					'type'  => 'custom',
+				),
+				array(
+					'title' => esc_html_x( 'Plugins', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/plugins/',
+					'type'  => 'custom',
+				),
+				array(
+					'title' => esc_html_x( 'Openverse', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/openverse/',
+					'type'  => 'custom',
+				),
+				array(
+					'title' => esc_html_x( 'Mobile', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/mobile/',
+					'type'  => 'custom',
+				),
+				array(
+					'title' => esc_html_x( 'Hosting', 'Menu item title', 'wporg' ),
+					'url'   => 'https://wordpress.org/hosting/',
+					'type'  => 'custom',
+				),
+			),
 		),
-
 		array(
-			'title' => esc_html_x( 'Support', 'Menu item title', 'wporg' ),
-			'url'   => 'https://wordpress.org/support/',
-			'type'  => 'custom',
-
+			'title'   => esc_html_x( 'Learn', 'Menu item title', 'wporg' ),
+			'url'     => 'https://learn.wordpress.org/',
+			'type'    => 'custom',
 			'submenu' => array(
 				array(
-					'title' => esc_html_x( 'Documentation', 'Menu item title', 'wporg' ),
+					'title' => esc_html_x( 'Support', 'Menu item title', 'wporg' ),
```

Comment:
I think we should maintain "Documentation" to refer to this https://wordpress.org/support/ page as it is ultimately a landing page for different documentation pages. 

I also think it would be confusing not having a "Documentation" link anywhere in the menu as I think this is a pretty much-expected link in any technical product site.